packages/plugin-pack/sources/packUtils.ts

```diff
@@ -69,7 +69,7 @@ export async function genPackStream(workspace: Workspace, files?: Array<Portable
   process.nextTick(async () => {
     for (const file of files!) {
       const source = ppath.resolve(workspace.cwd, file);
-      const dest = ppath.join(toFilename(`package`), file);
+      const dest = ppath.join(`package` as PortablePath, file);
```

Comment:
What's the rule of thumb for `toFilename` vs `as PortablePath`? The first for user-provided input and the second for literals that we know have the right format?

Reply:
IMO, `toFilename` should only be used as the last arguments in `ppath` or `npath`, when it represents a file (not a folder nor a part of a path). Otherwise, I'd use `PortablePath` 🙂

Reply:
Sounds right to me, using `toFilename` on folder names seems weird.

```ts
// That'd mean we'd get
const dest1 = ppath.join(toFilename(`package`), file);
// while could have
const dest2 = ppath.join(`packages/foo` as PortablePath, file);
// on the next line
```